pkg/models/query.go

```diff
@@ -92,30 +93,29 @@ func (f *TwinMakerPropertyFilter) ToTwinMakerFilter() *iottwinmaker.PropertyFilt
 	return filter
 }
 
-type TwinMakerOrderBy struct{
-	Order 		 TwinMakerResultOrder	`json:"order,omitempty"`
-	PropertyName string					`json:"propertyName,omitempty"`
+type TwinMakerOrderBy struct {
+	Order        TwinMakerResultOrder `json:"order,omitempty"`
+	PropertyName string               `json:"propertyName,omitempty"`
 }
 
 func (orderBy *TwinMakerOrderBy) ToTwinMakerOrderBy() *iottwinmaker.OrderBy {
-	o := &iottwinmaker.OrderBy{
-		Order: 		  &orderBy.Order,
-		PropertyName: &orderBy.PropertyName,
+	return &iottwinmaker.OrderBy{
+		Order:        aws.String(orderBy.Order),
```

Comment:
@hwandersman this was the main fix in this PR. it's a tricky part of looping with `range` in go: https://github.com/golang/go/discussions/56010